docs/source/diagnostics.rst

```diff
@@ -0,0 +1,86 @@
+Diagnostics
+===========
+
+Profiling parallel code can be tricky. ``dask.diagnostics`` provides
+functionality to aid in profiling and inspecting dask graph execution.
+
+
+Scheduler Callbacks
+-------------------
+
+Schedulers based on ``dask.async.get_async`` (currently
+``dask.async.get_sync``, ``dask.threaded.get``, and
+``dask.multiprocessing.get``) accept two callbacks, allowing for inspection of
+dask execution. The callbacks are:
+
+1. ``start_callback(self, key, dask, state)``
+    Callback ran every time a new task is started. Receives the key of the
+    task to be run, the dask, and the scheduler state. The final callback
+    receives ``None`` instead of a key, as no new tasks were added at that
+    tick.
```

Comment:
ran -> run (I think)

"The final callback" might not be clear to a naive user.  Suggest "This will also be called once at the end of the computation with `None` as the key."